tests/integration/test_api.py

```diff
@@ -0,0 +1,196 @@
+from __future__ import absolute_import, print_function
+import sys
+import os
+import uuid
+sys.path.append(os.path.join(os.path.dirname(os.path.realpath(__file__)), "..", ".."))
+
+from webserver.testing import ServerTestCase
+from flask import url_for
+import db.user
+import time
+import json
+
+# lifted from AcousticBrainz
+def is_valid_uuid(u):
```

Comment:
This is present in webserver.api_tools but is causing trouble when importing. I've copypastad it right now, but intend to see what's happening when importing from there.